client/src/api/treeTrackerApi.js

```diff
@@ -82,149 +75,176 @@ export default {
+      headers: {
+        Authorization: session.token,
+      },
+    })
+      .then(handleResponse)
+      .catch(handleError)
   },
   getTreeCount(filter) {
-    const query = `${process.env.REACT_APP_API_ROOT}/api/trees/count?${filter.getBackloopString(false)}`;
+    const query = `${process.env.REACT_APP_API_ROOT}/api/trees/count?${filter.getBackloopString(
+      false
+    )}`
     return fetch(query, {
-        headers: {
-          Authorization: session.token ,
-        }
-    }).then(handleResponse).catch(handleError);
+      headers: {
+        Authorization: session.token,
+      },
+    })
+      .then(handleResponse)
+      .catch(handleError)
   },
   getTreeById(id) {
-    const query = `${process.env.REACT_APP_API_ROOT}/api/trees/${id}`;
+    const query = `${process.env.REACT_APP_API_ROOT}/api/trees/${id}`
     return fetch(query, {
       headers: {
         Authorization: session.token,
-      }
-    }).then(handleResponse).catch(handleError);
+      },
+    })
+      .then(handleResponse)
+      .catch(handleError)
   },
   /*
    * get species list
    */
   getSpecies() {
-    const query = `${process.env.REACT_APP_API_ROOT}/api/species`;
+    const query = `${process.env.REACT_APP_API_ROOT}/api/species`
     return fetch(query, {
-      method: "GET",
-      headers: { 
-        "content-type": "application/json" ,
-        Authorization: session.token ,
+      method: 'GET',
+      headers: {
+        'content-type': 'application/json',
+        Authorization: session.token,
       },
     })
       .then(handleResponse)
-      .catch(handleError);
+      .catch(handleError)
   },
   /*
    * get species by id
    */
   getSpeciesById(id) {
-    const query = `${process.env.REACT_APP_API_ROOT}/api/species/${id}`;
+    const query = `${process.env.REACT_APP_API_ROOT}/api/species/${id}`
     return fetch(query, {
-      method: "GET",
-      headers: { 
-        "content-type": "application/json" ,
-        Authorization: session.token ,
+      method: 'GET',
+      headers: {
+        'content-type': 'application/json',
+        Authorization: session.token,
       },
     })
       .then(handleResponse)
-      .catch(handleError);
+      .catch(handleError)
   },
   /*
    * create new species
    */
   createSpecies(name) {
-    const query = `${process.env.REACT_APP_API_ROOT}/api/species`;
+    const query = `${process.env.REACT_APP_API_ROOT}/api/species`
     return fetch(query, {
-      method: "POST",
-      headers: { 
-        "content-type": "application/json" ,
-        Authorization: session.token ,
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json',
+        Authorization: session.token,
       },
       body: JSON.stringify({
         name: name,
         desc: name,
         active: 0,
         valueFactor: 0,
-      })
+      }),
+    })
+      .then(handleResponse)
+      .catch(handleError)
+  },
+  /* edit specie */
+  editSpecie(id, name, desc) {
```

Comment:
Should be `editSpecies` (singular of species is species)

Reply:
lol thanks! That's embarrassing haha why does it not have a pattern 👍